captainhook.json

```diff
@@ -16,17 +16,17 @@
                 "conditions": []
             },
             {
-                "action": "vendor/bin/ecs check --ansi",
+                "action": "composer run check-style",
```

Comment:
Do the composer changes need to be made in this commit? So you have a dependency on the rector? Otherwise it would be a separate commit for me.

Reply:
It is a separate commit.